src/Symfony/Component/Notifier/Bridge/Telegram/Reply/Markup/AbstractTelegramReplyMarkup.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Telegram\Reply\Markup;
+
+/**
+ * @author Mihail Krasilnikov <mihail.krasilnikov.j@gmail.com>
+ */
```

Comment:
You should add `@experimental in 5.2` in all new classes.

Reply:
Added, sorry ma bad.